src/openrct2-ui/windows/About.cpp

```diff
@@ -202,55 +202,54 @@ static void window_about_openrct2_common_paint(rct_window* w, rct_drawpixelinfo*
 {
     window_draw_widgets(w, dpi);
 
-    int32_t x1, x2, y;
-
-    x1 = w->windowPos.x + (&w->widgets[WIDX_TAB_ABOUT_OPENRCT2])->left + 45;
-    x2 = w->windowPos.x + (&w->widgets[WIDX_TAB_ABOUT_RCT2])->left + 45;
-    y = w->windowPos.y + (((&w->widgets[WIDX_TAB_ABOUT_OPENRCT2])->top + (&w->widgets[WIDX_TAB_ABOUT_OPENRCT2])->bottom) / 2)
-        - 3;
+    int32_t y =
+        w->windowPos.y + (((&w->widgets[WIDX_TAB_ABOUT_OPENRCT2])->top + (&w->widgets[WIDX_TAB_ABOUT_OPENRCT2])->bottom) / 2) - 3;
```

Comment:
This is pretty horrible to read. Could you create a temporary ABOUT_OPENRCT2 widget something like:

```
auto aboutWidget = &w->widgets[WIDX_TAB_ABOUT_OPENRCT2];
y = w->windowPos.y + (aboutWidget.top + aboutWidget.bottom) / 2 - 3;
ScreenCoordsXY coord1(w->windowPos.x + aboutWidget.left + 45, y);
```